java-checks/src/test/files/checks/security/ZipEntryCheck.java

```diff
@@ -0,0 +1,52 @@
+package org.foo;
+
+import java.util.Enumeration;
+import java.util.jar.JarEntry;
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipFile;
+import org.apache.commons.compress.archivers.ArchiveEntry;
+import org.apache.commons.compress.archivers.zip.ZipArchiveEntry;
+
+abstract class A {
```

Comment:
minor detail but for sake of documentation : having one non compliant and a compliant cases with fields (not only method parameters and local vars).